src/main/java/io/confluent/connect/jdbc/sink/dialect/VerticaDialect.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.connect.jdbc.sink.dialect;
+
+import org.apache.kafka.connect.data.Date;
+import org.apache.kafka.connect.data.Decimal;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Time;
+import org.apache.kafka.connect.data.Timestamp;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import io.confluent.connect.jdbc.sink.metadata.SinkRecordField;
+
+public class VerticaDialect extends DbDialect {
+  public VerticaDialect() {
+    super("\"", "\"");
+  }
+
+  @Override
+  protected String getSqlType(String schemaName, Map<String, String> parameters, Schema.Type type) {
+    if (schemaName != null) {
+      switch (schemaName) {
+        case Decimal.LOGICAL_NAME:
+          return "DECIMAL(18," + parameters.get(Decimal.SCALE_FIELD) + ")";
```

Comment:
For most DBs the question here is what the range of valid values is. 18 is unusually small. Something like 38 or 65 pop up commonly (see the table that you updated in the README). I think the question here will be whether optimizing for the common case (probably fits within p=18) or covering all cases (e.g. p=67 given what databases generally cover) is the right choice.